frontend/src/components/print/print-react/i18n.js

```diff
@@ -1,8 +1,8 @@
 import {
-  translate,
+  compileToFunction,
```

Comment:
what happened here?

Reply:
optimized imports in the IDE, which sorted them alphabetically